code/modules/karma/karma.dm

```diff
@@ -117,6 +120,20 @@ var/list/karma_spenders = list()
 
 	spend_karma(pickedmob)
 
+/mob/verb/toggle_karma()
+	set name = "Toggle Karma Gains"
+	set desc = "Unlocked everything? Got too much karma? This button will let you disable karma gains!"
+	set category = "Special Verbs"
+    if(prefs.Toggle & KARMA_TOGGLE)
+        client.wants_karma = FALSE
+        to_chat(usr, "<span class='notice'>You have disabled karma gains.")
+    else if(!client.wants_karma)
```

Comment:
Why do you check "prefs.Toggle & KARMA_TOGGLE" for the first check, and "else if(!client.wants_karma)" for the second? Surely the second check could just be 'else'? It is a boolean, there are only two possible states?